governance/xc-admin/packages/xc-admin-frontend/pages/index.tsx

```diff
@@ -1,13 +1,99 @@
+import { Tab } from '@headlessui/react'
 import type { NextPage } from 'next'
+import { useRouter } from 'next/router'
+import { useEffect, useState } from 'react'
 import Layout from '../components/layout/Layout'
-import MinPublishers from '../components/MinPublishers'
+import MinPublishers from '../components/tabs/MinPublishers'
+import UpdatePermissions from '../components/tabs/UpdatePermissions'
 import { PythContextProvider } from '../contexts/PythContext'
+import { classNames } from '../utils/classNames'
+
+const tabInfo = {
+  MinPublishers: {
+    title: 'Min Publishers',
+    description:
+      'Set the minimum number of publishers required to publish a price.',
+    queryString: 'min-publishers',
+  },
+  UpdatePermissions: {
+    title: 'Update Permissions',
+    description: 'Update the permissions of the program.',
+    queryString: 'update-permissions',
+  },
+}
+
+const DEFAULT_TAB = 'min-publishers'
 
 const Home: NextPage = () => {
+  const [currentTabIndex, setCurrentTabIndex] = useState(0)
```

Comment:
Does this need to be a number, wouldn't it make more sense as a "updatePermissions" | "setMinPub" or a string

Reply:
yes -- we are using Tabs from @headless-ui and they require index to be passed in for their onChange handler